payments-core/src/main/java/com/stripe/android/model/LinkMode.kt

```diff
@@ -4,6 +4,19 @@ import androidx.annotation.RestrictTo
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 enum class LinkMode(val value: String) {
-    Passthrough("passthrough"),
-    PaymentMethod("payment_method_mode"),
+    Passthrough("PASSTHROUGH"),
+    LinkPaymentMethod("LINK_PAYMENT_METHOD"),
+    LinkCardBrand("LINK_CARD_BRAND"),
```

Comment:
What does this mode do? My initial assumption is that it is for Card Brand Choice which we added [popup parameters](https://github.com/stripe/stripe-android/pull/9222) for recently.

Reply:
It’ll be used [here](https://github.com/stripe/stripe-android/pull/9288/files#diff-d2df7ce550b5a59fef8971c747068f960fc8d84fd77cdd1641e627507e185527R154) to add the Link card brand payment method.